src/screens/sync/DevicesList.tsx

```diff
@@ -34,16 +34,18 @@ export const DevicesList = ({
   peers,
   onDevicePress,
   onInfoPress,
+  show,
 }: {
   peers: Peer[];
   onDevicePress: (isNotIdle: boolean, peer: Peer) => void;
   onInfoPress: () => void;
+  show: boolean;
 }) => {
   const { formatMessage: t } = useIntl();
   const [shouldStart, setShouldStart] = React.useState(false);
 
   return (
-    <View>
+    <View style={show ? undefined : { display: "none" }}>
```

Comment:
The progress state is held locally. So everytime we unmounted this (everytime we opened the bubbles page), it caused the  sync to start again. This is hacky, but we can't lift the state of the progress without having performance issues.

Reply:
ah was about to comment about this but makes sense. thanks for clarifying!